TileFormats/FeatureTable/schema/featureTable.schema.json

```diff
@@ -0,0 +1,91 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema#",
+    "id" : "featureTable.schema.json",
+    "title" : "Feature Table",
+    "type" : "object",
+    "description" : "A set of semantics containing per-tile and per-feature values defining the position and appearance properties for features in a tile.",
+    "patternProperties" : {
+        ".*" : {
+            "properties" : {
+                "byteOffset" : {
+                    "type" : "integer",
+                    "minimum" : 0
+                }
+            }
+        }
+    },
+    "definitions" : {
+        "featureReference" : {
+            "type" : "object",
+            "required" : ["byteOffset"]
+        },
+        "featureArray" : {
+            "type" : "array",
+            "items" : {
+                "type" : "number"
```

Comment:
For the base Feature Table spec, we can't require that arrays only contain numbers that would prevent new tile formats from using other datatypes.